eva_submission/eload_ingestion.py

```diff
@@ -0,0 +1,111 @@
+import string
+import subprocess
+
+from bs4 import BeautifulSoup
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.mongo_utils import get_mongo_connection_handle
+import requests
+
+from eva_submission.eload_submission import Eload
+
+
+def sanitize(s):
+    """Lowercases and removes punctuation from s, e.g. to make it a legal MongoDB identifier."""
+    return s.lower().translate(str.maketrans('', '', string.punctuation))
```

Comment:
Wouldn't it be a better idea to disallow anything but alphanumeric and underscore?

Reply:
This isn't necessary anymore as we'll get the appropriate codes from evapro as discussed.